src/bridge/cockpitpackages.c

```diff
@@ -990,7 +990,17 @@ package_content (CockpitPackages *packages,
         {
           /* On the first round */
           if (l == names)
-            cockpit_web_response_error (response, 404, NULL, NULL);
+            {
+              /* cockpit_packages_resolve() only fails if the entire
+               * package is missing.  Check if that's a package that
+               * ought to have been available and issue a more helpful
+               * message.
+               */
+              if (g_str_equal (name, "shell") || g_str_equal (name, "systemd"))
+                cockpit_web_response_error (response, 404, NULL, "Server is missing the cockpit-system package");
```

Comment:
Nice and simple! But let's please wrap this in `_()` for translation.

Reply:
done, thanks.

I assume cockpit-bridge will be running in a session with the locale correctly set as per the browser locale?